src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -47,6 +47,14 @@
     <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="runtime" GeneratePathProperty="true" />
   </ItemGroup>
 
+  <ItemGroup Label="transitive package pinning">
+    <!--
+      These packages are dependencies of directly referenced PackageReferences.
+      When the above PackageReferences are upgraded to newer versions, try deleting the below PackageReferences
+      -->
+    <PackageReference Include="System.Security.Cryptography.Xml" GeneratePathProperty="true" Condition=" '$(TargetFramework)' != '$(NETFXTargetFramework)' " />
```

Comment:
Where is this one coming from?

Reply:
```
> dotnet nuget why .\src\NuGet.Core\NuGet.Build.Tasks.Pack\ System.Security.Cryptography.Xml
Project 'NuGet.Build.Tasks.Pack' has the following dependency graph(s) for 'System.Security.Cryptography.Xml':

  [netstandard2.0]
   │
   ├─ Microsoft.Build.Tasks.Core (v16.8.0)
   │  └─ System.Security.Cryptography.Xml (v8.0.1)
   └─ System.Security.Cryptography.Xml (v8.0.1)

  [net472]
   │
   └─ No dependency graph(s) found for this target framework.
```

Reply:
Ah, and we can't move it cause of our target frameworks not being compatible with newer MSBuild.

We really need https://github.com/NuGet/Home/issues/13079 so we can change the TFM of our project here.